src/api_manager/BUILD

```diff
@@ -302,20 +302,19 @@ cc_test(
     ],
 )
 
-# Disabled due to flaky test in opt mode
-# cc_test(
-#     name = "config_manager_test",
-#     size = "small",
-#     srcs = [
-#         "config_manager.h",
-#         "config_manager_test.cc",
-#     ],
-#     deps = [
-#         ":api_manager",
-#         ":mock_api_manager_environment",
-#         "//external:googletest_main",
-#     ],
-# )
+cc_test(
```

Comment:
That is not part of the PR. These are changed files included .

$ git status
On branch master
Your branch is ahead of 'origin/master' by 2 commits.
  (use "git push" to publish your local commits)

Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git checkout -- <file>..." to discard changes in working directory)

	modified:   include/api_manager/api_manager.h
	modified:   src/api_manager/api_manager_impl.cc
	modified:   src/api_manager/api_manager_impl.h
	modified:   src/api_manager/weighted_selector.h
	modified:   src/nginx/proto/status.proto
	modified:   src/nginx/status.cc

It seems that meringue back from master included old commits, but those are not part of this.